CodePush.js

```diff
@@ -271,13 +271,9 @@ async function syncInternal(options = {}, syncStatusChangeCallback, downloadProg
             log("User cancelled the update.");
             break;
           case CodePush.SyncStatus.UPDATE_INSTALLED:
-            /*
-             * If the install mode is IMMEDIATE, this will not get returned as the
-             * app will be restarted to a new Javascript context.
-             */
             if (resolvedInstallMode == CodePush.InstallMode.ON_NEXT_RESTART) {
               log("Update is installed and will be run on the next app restart.");
-            } else {
+            } else if (resolvedInstallMode == CodePush.InstallMode.ON_NEXT_RESUME) {
```

Comment:
During my testing, I found that for an IMMEDIATE install, even though the JS context gets reloaded immediately, the "resume logging" still shows up in the IDE console, that's why I made this change.